extensions/stackdriver/metric/registry.cc

```diff
@@ -80,16 +81,19 @@ getStackdriverOptions(const Wasm::Common::FlatNode& local_node_info,
     }
   }
 
-  auto ssl_creds_options = grpc::SslCredentialsOptions();
+  grpc::experimental::TlsChannelCredentialsOptions tls_options;
```

Comment:
Based on its name, "grpc::experimental::TlsChannelCredentialsOptions" seems to be an experimental feature. Do we have a more stable API to set the max TLS version?  If this experimental feature becomes deprecated, we need to find a replacement for it.

Reply:
gRPC team specifically backported this experimental feature for us. A similar change in Envoy was approved https://github.com/envoyproxy/envoy/pull/32315.